lib/advent_of_code/input.ex

```diff
@@ -0,0 +1,87 @@
+defmodule AdventOfCode.Input do
+  @moduledoc """
+  This module can help with automatically managing your Advent of Code input
+  files. It will retrieve them once from the server and cache them to your
+  machine.
+  By default, it is configured to have network requests disabled. You can
+  easily turn it on by editing the configuration.
+  """
+
+  import Logger
+
+  @doc """
+  Retrieves the specified input for your account. If the input is not in your
+  cache, it will be retrieved from the server if `allow_network?: true` is
+  configured and your cookie is setup.
+  """
+  def get!(day, year \\ nil)
+  def get!(day, nil), do: get_input!(day, default_year())
```

Comment:
this this be just `get!(day, default_year())`? @lytedev

Reply:
Oops yes.

Reply:
No worries, I've got it fixed locally 👍